src/main/java/socialite/model/handle/Facebook.java

```diff
@@ -0,0 +1,54 @@
+package socialite.model.handle;
+
+import static java.util.Objects.requireNonNull;
+
+import socialite.commons.util.AppUtil;
+
+/**
+ * Represents a Person's Facebook username in SociaLite.
+ * Guarantees: immutable; handle is valid as declared in {@link #isValidHandle(String)}
+ */
+public class Facebook {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "A Facebook username should only contain alphanumeric characters and full stops."
```

Comment:
Maybe a space or newline can be added here.

Reply:
Thanks for reviewing the codes @bnjmnt4n! I've adopted your above suggestions in my latest commits and included the Facebook icon in the UI as well. I'll do the same for the Instagram handles.